mapit-gb/urls.py

```diff
@@ -6,4 +6,5 @@
     url(r'^', include('mapit.urls')),
     url(r'^admin/', include(admin.site.urls)),
     url(r"^account/", include("account.urls")),
+    url(r"^account/api_keys/", include("api_keys.urls")),
```

Comment:
This is presumably fine, as the accounts app presumably doesn't have an `api_keys/` entry, but putting it above `^account/` would probably be clearer.